manifests/machineconfig.crd.yaml

```diff
@@ -39,205 +41,401 @@ spec:
                       type: string
+                networkd:
+                  type: object
+                  properties:
+                    units:
+                      type: array
+                      items:
+                        type: object
+                        properties:
+                          contents:
+                            type: string
+                          dropins:
+                            type: array
+                            items:
+                              type: object
+                              properties:
+                                contents:
+                                  type: string
+                                name:
+                                  type: string
+                          name:
+                            type: string
                 passwd:
                   type: object
                   properties:
+                    groups:
+                      type: array
+                      items:
+                        type: object
+                        properties:
+                          gid:
+                            type: integer
+                          name:
+                            type: string
+                          passwordHash:
+                            type: string
+                          system:
+                            type: boolean
                     users:
                       type: array
                       items:
                         type: object
                         properties:
+                          create:
+                            type: object
+                            properties:
+                              gecos:
+                                type: string
+                              groups:
+                                type: array
+                                items:
+                                  type: string
+                              homeDir:
+                                type: string
+                              noCreateHome:
+                                type: boolean
+                              noLogInit:
+                                type: boolean
+                              noUserGroup:
+                                type: boolean
+                              primaryGroup:
+                                type: string
+                              shell:
+                                type: string
+                              system:
+                                type: boolean
+                              uid:
+                                type: integer
+                          gecos:
+                            type: string
+                          groups:
+                            type: array
+                            items:
+                              type: string
+                          homeDir:
+                            type: string
```

Comment:
We don’t support all these ignition fields. I think it’s only the ones we had that we allow

Reply:
Same as above, I will look to manually edit for now, unless you know of a better method we can source these MCO-specific use-cases.